src/main/java/io/moderne/organizations/OrganizationStructureService.java

```diff
@@ -190,6 +190,12 @@ private static String cleanPath(String path) {
             if (path.startsWith("/")) {
                 path = path.substring(1);
             }
+            // In case of bitbucket server/on prem we need to remove the `/scm` prefix.
+            // his prefix is not part of all URL's to repository resource
+            // (for instance pull requests) so it cannot be part of the origin or path.
+            if (path.startsWith("scm/")) {
+                path = path.substring(4);
+            }
```

Comment:
We talked about this a little in Slack, but essentially if someone has `github.mycompany.com/scm/` as their root, it would remove `scm` even though it shouldn't?

I think the likelihood of this is low, but we could get around it by adding an "scm type" into scm-origins (and make it either a properties file or CSV).

Reply:
https://github.com/moderneinc/moderne-organizations/pull/84